src/Utils/Custom/Custom.ts

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 
```

Comment:
The name of the file doesn't describe that it will contains the base URL or something similar